app/qml/main.qml

```diff
@@ -430,6 +432,8 @@ ApplicationWindow {
                 digitizingHighlight.featureLayerPair = digitizing.featureLayerPair
             }
         }
+
+        onUseGpsPointChanged: __variablesManager.useGpsPoint = digitizing.useGpsPoint
```

Comment:
why not direct binding?

Reply:
I think another solution can be defining QML `Binding` since  __variablesManager is created outside of QML.
Or just listen on change signal.